OsmAnd-java/src/main/java/net/osmand/GPXUtilities.java

```diff
@@ -507,6 +511,28 @@ private void updatePoint(double lat, double lon, String description, String name
 				setBackgroundType(backgroundType);
 			}
 		}
+		
+		private void updatePoint(double lat, double lon, String description, String name,
```

Comment:
duplicate method

Reply:
this is overload method from android. i can't change safely such methods